src/components/FileInput/index.jsx

```diff
@@ -69,12 +69,10 @@ class FileInput extends React.Component {
     } = this.props;
     const hasError = !!(touched && error);
     return (
-      <ValidationFormGroup
+      <Form.Group
```

Comment:
[suggestion, non-blocking] Perhaps for a separate PR/ticket, but I wonder if it might be worth considering refactoring this component to use Paragon's `Dropzone` component ([docs](https://paragon-openedx.netlify.app/components/dropzone/)) 🤔

Reply:
Good idea! https://2u-internal.atlassian.net/browse/ENT-8430